packages/react-data-grid/src/Cell/CellContent.tsx

```diff
@@ -69,18 +69,16 @@ export default function CellContent<R>({
   return (
     <div className={classes}>
       {cellDeleter}
-      <div className="react-grid-Cell__container" style={{ marginLeft }}>
-        <span>
-          <CellValue<R>
-            rowIdx={rowIdx}
-            rowData={rowData}
-            column={column}
-            value={value}
-            isScrolling={isScrolling}
-            isRowSelected={isRowSelected}
-            onRowSelectionChange={onRowSelectionChange}
-          />
-        </span>
+      <div style={style}>
+        <CellValue<R>
+          rowIdx={rowIdx}
+          rowData={rowData}
+          column={column}
+          value={value}
+          isScrolling={isScrolling}
+          isRowSelected={isRowSelected}
+          onRowSelectionChange={onRowSelectionChange}
+        />
         {cellControls}
       </div>
       {tooltip && <span className="cell-tooltip-text">{tooltip}</span>}
```

Comment:
https://github.com/adazzle/react-data-grid/pull/1773/files/cba04c56c295e32fa3f036cb6cd9dbd5f8d5ec60#r333000852

Thoughts about removing this here? Since the style is touched, if we want to remove, I bet it's better in this PR. Or we keep it for revisiting.

Reply:
Let's do it in a separate PR.